kml/serdes_common.hpp

```diff
@@ -21,4 +24,16 @@ std::string PointToString(m2::PointD const & org);
 
 std::string PointToString(geometry::PointWithAltitude const & pt);
 
+void SaveStringWithCDATA(Writer & writer, std::string s);
+
+namespace
+{
+std::string_view constexpr kIndent0 = {};
```

Comment:
nit: remove =, less characters looks cleaner. type{} means to call a constructor, similar to type(), but with a stricter parameter checks. With a benefit that it's can not be read as a function call.

Reply:
Fixed